apps/meteor/app/api/server/v1/teams.ts

```diff
@@ -85,7 +85,17 @@ API.v1.addRoute(
 				}),
 			);
 
-			const { name, type, members, room, owner } = this.bodyParams;
+			const { name, type, members, room, owner, sidepanel } = this.bodyParams;
+
+			if (
+				sidepanel?.items &&
+				(!Array.isArray(sidepanel.items) ||
+					!sidepanel.items.length ||
+					!sidepanel.items.every(isSidepanelItem) ||
+					sidepanel.items.length > 2)
+			) {
+				throw new Error('error-invalid-sidepanel');
+			}
```

Comment:
Can you please add some end-to-end tests to validate that this is working fine? And also for the case when it should be accepted

Reply:
done! thanks 😄